tests/php/test_deprecation.php

```diff
@@ -28,17 +38,46 @@ public function assertDeprecatedFileError( $deprecated, $replacement, $errno ) {
 		$this->fail( "Error for $deprecated not found" );
 	}
 
-	function test_require_old_jetpack_options() {
-		$this->set_error_handler();
-		require_once JETPACK__PLUGIN_DIR . '/class.jetpack-options.php';
+	/**
+	 * @dataProvider provider_deprecated_file_paths
+	 */
+	function test_deprecated_file_paths( $file_path, $replacement_path, $error_level ) {
+		require_once JETPACK__PLUGIN_DIR . $file_path;
 
-		$this->assertDeprecatedFileError( 'class.jetpack-options.php', 'packages/options/legacy/class.jetpack-options.php',
-			E_USER_NOTICE );
-		restore_error_handler();
+		$this->assertDeprecatedFileError( $file_path, $replacement_path, $error_level );
+	}
+
+	/**
+	 * @dataProvider provider_deprecated_method_stubs
+	 */
+	function test_deprecated_method_stubs( $class_name, $method_name ) {
+		$this->assertTrue( method_exists( $class_name, $method_name ) );
+	}
+
+	function test_jetpack_client__remote_request() {
```

Comment:
Did you mean to remove this?

Reply:
Of course, thank you, will remove it now.